channels/fast-4.4.yaml

```diff
@@ -2,8 +2,6 @@ name: fast-4.4
 versions:
 - 4.3.19
 
```

Comment:
Is this empty line intended?

Reply:
Art usually adding empty lines after each entry. They can clarify. cc  @jupierce